lib/dom-parser.js

```diff
@@ -49,98 +43,82 @@ function normalizeLineEndings(input) {
+ * prevents any further processing. If the provided method throws, a `ParserError` is thrown, which prevents any further
+ * processing.
  *
- * Be aware that many `warning`s are considered an error
- * that prevents further processing in most implementations.
- *.
+ * Be aware that many `warning`s are considered an error that prevents further processing in most implementations. .
  * @property {boolean} [locator=true]
- * Configures if the nodes created during parsing
- * will have a `lineNumber` and a `columnNumber` attribute
- * describing their location in the XML string.
- * Default is true.
+ * Configures if the nodes created during parsing will have a `lineNumber` and a `columnNumber`
+ *
+ * Attribute describing their location in the XML string. Default is true. Default is `true`
  * @property {(string) => string} [normalizeLineEndings]
- * used to replace line endings before parsing, defaults to `normalizeLineEndings`
- * @property {object} [xmlns]
- * The XML namespaces that should be assumed when parsing.
- * The default namespace can be provided by the key that is the empty string.
- * When the `mimeType` for HTML, XHTML or SVG are passed to `parseFromString`,
- * the default namespace that will be used,
+ * Used to replace line endings before parsing, defaults to `normalizeLineEndings`
+ * @property {Object} [xmlns]
+ * The XML namespaces that should be assumed when parsing. The default namespace can be provided by the key that is the empty
+ * string. When the `mimeType` for HTML, XHTML or SVG are passed to `parseFromString`, the default namespace that will be used,
  * will be overridden according to the specification.
- *
- * @see normalizeLineEndings
+ * @see NormalizeLineEndings.
  */
 
 /**
- * The DOMParser interface provides the ability to parse XML or HTML source code
- * from a string into a DOM `Document`.
+ * The DOMParser interface provides the ability to parse XML or HTML source code from a string into a DOM `Document`.
  *
- * _xmldom is different from the spec in that it allows an `options` parameter,
- * to control the behavior._
+ * _xmldom is different from the spec in that it allows an `options` parameter, to control the behavior._.
  *
+ * @class
  * @param {DOMParserOptions} [options]
- * @constructor
- *
  * @see https://developer.mozilla.org/en-US/docs/Web/API/DOMParser
  * @see https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#dom-parsing-and-serialization
  */
 function DOMParser(options) {
 	options = options || { locator: true };
 
 	/**
-	 * The method to use instead of `Object.assign` (defaults to or `conventions.assign`),
-	 * which is used to copy values from the options before they are used for parsing.
+	 * The method to use instead of `Object.assign` (defaults to or `conventions.assign`), which is used to copy values from the
+	 * options before they are used for parsing.
```

Comment:
Should also be synced with the comment suggestion from above: 
```suggestion
	 * The method to use instead of `conventions.assign`, which is used to copy values from `options` before they are used for parsing.
   * Default is `conventions.assign`
```